src/RpcClient/Utility.cs

```diff
@@ -205,6 +205,8 @@ public static StackItem StackItemFromJson(JObject json)
             StackItemType type = json["type"].TryGetEnum<StackItemType>();
             switch (type)
             {
+                case StackItemType.Any:
+                    return StackItem.Null;
```

Comment:
I think that `null` is `Any` but any is not `null` , what do you think @erikzhang ?

Reply:
Agree.

Reply:
I changed the code.